openapi/components/schemas/GatewayAccount.yaml

```
@@ -369,6 +369,29 @@ properties:
       do-nothing: |-
         Does nothing except return an approved `setup` transaction.
         This is the default behavior.
+  readyToPayoutInstruction:
+    type:
+      - 'string'
+      - 'null'
+    default: none
+    description: |-
+      Defines a mode of gateway payout behaviour for [StorefrontReadyToPayout](https://all-rebilly.redoc.ly/tag/Purchases/operation/StorefrontPostReadyToPayout).
```

suggestion: for readability

I get a 404 when I try to visit the list [the URL](https://all-rebilly.redoc.ly/tag/Purchases/operation/StorefrontPostReadyToPayout)

```suggestion
      Defines the mode of gateway payout behaviour for the [Ready to payout](https://all-rebilly.redoc.ly/tag/Purchases/operation/StorefrontPostReadyToPayout) operation.
```